storage/src/mmr/bitmap.rs

```diff
@@ -0,0 +1,296 @@
+//! An authenticatable bitmap.
```

Comment:
Wondering if this may be better placed as a standalone "dialect" (storage/src/bitmap)?

Reply:
Yeah I had debated that... Happy to pull it out?

Reply:
My thinking was maybe leave it here for now and consider moving it out with a broader refactor once the mutable store is functional.

Reply:
Sounds good. All for getting it "all to work" before making these sorts of decisions.